src/xunit.performance.api/xunit.performance.api.csproj

```diff
@@ -20,16 +20,13 @@
     <Compile Include="..\common\GlobalAssemblyInfo.cs" Link="GlobalAssemblyInfo.cs" />
   </ItemGroup>
 
-  <PropertyGroup>
-    <NetStandardImplicitPackageVersion>1.6.1</NetStandardImplicitPackageVersion>
```

Comment:
It is inferred from here: `<TargetFramework>netstandard1.5</TargetFramework>`.
I was able to cleanup the csproj, and I'm not building the old project anymore. Thus, I was able to remove building net46 and some warnings we used to get.